### ISSUE TITLE
Subplots

### ISSUE THREAD

**Issue by mbostock** [observablehq/plot#1870]
The “subplot” mark would essentially be a faceted plot specification within an existing plot, but with facets positioned arbitrarily by an anchor in *x* and *y*, and with a fixed *width* and *height*. The subplots would share the same scales.

**Comment by Hvass-Labs** [comment]
> "The subplots would share the same scales."

I would very much like if I could have different scales (I guess that means axis-ranges?) in the sub-plots. The way this is done in Matplotlib is to specify whether the sub-plots should share the x and y axis ranges. Perhaps this would be possible here as well? Thanks!

**Comment by awensaunders** [comment]
I agree that it would be really useful to have subplots with different scales (or at least one different axis for my use case).

For example, I want to have several plots of time series data with over the same time scale as the x axis, but each with a different y axis. This is an alternative to having a plot with two (or more) y axis scales which is a common anti pattern in data vis.

**Comment by Fil** [comment]
Here's a possible approach
https://observablehq.com/@fil/subplots-1870
